### PR TITLE
Removed lombok dependency

### LINKED ISSUE #386: [Core] Remove Lombok
Remove the `org.projectlombok` dependencies and usage in XSK, reason - unsustainable usage.

### PR DESCRIPTION
**Pull request by aihanrashidov**
Resolves #386 

- removed `lombok` dependency from pom files
- removed `lombok` imports from classes
- changed some variable names and added`@SerializedName` annotation where needed
- added setters and getters
- added `constructor`, `equals` and `hashCode` where needed depending on previously used `lombok` annotations

### CODE REVIEW

File: modules/parsers/parser-hdbsequence/src/main/java/com/sap/xsk/parser/hdbsequence/models/XSKHDBSEQUENCEModel.java [hunk 1]
```diff
@@ -13,39 +13,169 @@
...
-  String depends_on_view;
+
+  @SerializedName(value = HDBSequenceConstants.RESET_BY_PROPERTY)
+  String resetBy;
+
+  @SerializedName(value = HDBSequenceConstants.DEPENDS_ON_TABLE_PROPERTY)
+  String dependsOnTable;
+
+  @SerializedName(value = HDBSequenceConstants.DEPENDS_ON_VIEW_PROPERTY)
+  String dependsOnView;
```
Comment (ThuF) [inline modules/parsers/parser-hdbsequence/src/main/java/com/sap/xsk/parser/hdbsequence/models/XSKHDBSEQUENCEModel.java:55]: Add the **private** modifier?

File: modules/parsers/parser-hdbti/src/main/java/com/sap/xsk/parser/hdbti/models/XSKHDBTIImportConfigModel.java [hunk 2]
```diff
@@ -13,88 +13,204 @@
...
-@Getter
-@Setter
-@NoArgsConstructor
 public class XSKHDBTIImportConfigModel {
 
-    @SerializedName("table")
-    @JsonProperty("table")
-    private String tableName;
+  @SerializedName("table")
+  @JsonProperty("table")
```
Comment (ThuF) [inline modules/parsers/parser-hdbti/src/main/java/com/sap/xsk/parser/hdbti/models/XSKHDBTIImportConfigModel.java:23]: Do we need the both annotations? Can't we use only **@SerializedName**?

**Review by ThuF — CHANGES_REQUESTED** [review]
Few small comments from my side.

Waiting for @antoanetamh for the final approval of this PR.

**Review by ThuF — APPROVED** [review]
+1 Waiting for @antoanetamh for the final approval of this PR.

File: modules/parsers/parser-hdbti/src/main/java/com/sap/xsk/parser/hdbti/models/XSKHDBTIImportConfigModel.java [hunk 3]
```diff
@@ -11,90 +11,196 @@
...
 
-@Getter
-@Setter
-@NoArgsConstructor
 public class XSKHDBTIImportConfigModel {
 
-    @SerializedName("table")
-    @JsonProperty("table")
-    private String tableName;
+  @SerializedName("table")
```
Comment (antoanetamh) [inline modules/parsers/parser-hdbti/src/main/java/com/sap/xsk/parser/hdbti/models/XSKHDBTIImportConfigModel.java:21]: Why @JsonProperty was removed?

Comment (ThuF) [reply modules/parsers/parser-hdbti/src/main/java/com/sap/xsk/parser/hdbti/models/XSKHDBTIImportConfigModel.java:21]: @antoanetamh it was replaced by GSON serialization with the `@SerializedName` annotation.

Comment (antoanetamh) [reply modules/parsers/parser-hdbti/src/main/java/com/sap/xsk/parser/hdbti/models/XSKHDBTIImportConfigModel.java:21]: Correct, if we are using Gson.
https://www.baeldung.com/json-multiple-fields-single-java-field

**Comment by antoanetamh** [comment]
The PR can be merged ;)

**Pull request merged**